exports/taskfiles/utils/misc.yaml

```diff
@@ -41,3 +43,47 @@ tasks:
         . "{{.OUTPUT_DIR}}/bin/activate"
         pip3 install --upgrade pip
         pip3 install --upgrade -r "{{.REQUIREMENTS_FILE}}"
+
+  reset-dir:
+    desc: "Clears the directory at the given path without removing the directory itself."
+    internal: true
+    label: "{{.TASK}}:{{.DIR_PATH}}"
+    requires:
+      vars: ["DIR_PATH"]
+    run: "when_changed"
+    preconditions:
+      # This task runs once per unique directory path, which is fingerprinted by the task label. To
+      # avoid possible label collisions, we avoid using relative paths.
+      - sh: >-
+          case "{{.DIR_PATH}}" in
+            /*) exit 0;;
+            *) exit 1;;
+          esac
+        msg: "DIR_PATH `{{.DIR_PATH}}` is not absolute."
+    cmds:
+      - |-
+        mkdir -p "{{.DIR_PATH}}"
+        find "{{.DIR_PATH}}" \! -path "{{.DIR_PATH}}" -prune \
+          -exec rm -rf -- {} +
+
+  reset-file:
+    desc: "Removes the file at the given path and makes sure its parent directory exists."
+    internal: true
+    label: "{{.TASK}}:{{.FILE_PATH}}"
+    vars:
+      PARENT_DIR: "{{ dir .FILE_PATH }}"
+    requires:
+      vars: ["FILE_PATH"]
+    run: "when_changed"
+    preconditions:
+      # This task runs once per unique file path, which is fingerprinted by the task label. To avoid
+      # possible label collisions, we avoid using relative paths.
```

Comment:
Similar comments about doc strings and the name as above.

In this case the name is more serious. If the file doesn't already exist, it will not be created (so it won't be "reset", it just still won't exist) and potentially any parent directories will be created.